RxSession.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'RxSession'
-  s.version          = '0.2.0'
+  s.version          = '1.0.0'
```

Comment:
In Zeile 39 solltest du auf Version 2.0.0 überprüfen von Endpoints

Reply:
done